pkgs/development/python-modules/pysunspec2/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  pythonOlder,
+  fetchFromGitHub,
+  buildPythonPackage,
+  pytestCheckHook,
+  setuptools,
+  pyserial,
+  openpyxl,
+}:
+
+buildPythonPackage rec {
+  pname = "pysunspec2";
+  version = "1.2.1";
```

```suggestion
  version = "1.2.1";
  pyproject = true;
```
The default is sadly still `false`